gstd/gstd.c

```diff
@@ -174,6 +181,15 @@ main (gint argc, gchar * argv[])
     {"no-daemon", 'D', 0, G_OPTION_ARG_NONE, &nodaemon,
         "Do not detach into a daemon", NULL}
     ,
+    {"pid-path", 'P', 0, G_OPTION_ARG_FILENAME, &pidfile,
```

Comment:
Use `p` instead of `P`. It's common to Upper case chars for negated options, like for example `no-daemon/D`